src/components/LoginForm/PersonalFormFields.tsx

```diff
@@ -55,7 +55,7 @@ export default function PersonalFormFields({ user }: { user: AuthUser }) {
   };
   const updateSelectInput = (e: React.ChangeEvent<HTMLSelectElement>): void => {
     const { name, value } = e.target;
-    if (name === "meals") {
+    if (name === "willEatMeals") {
```

Comment:
must be willEatMeals

Reply:
Does this get translated to true false somewhere? Because when we submit we want to submit true, not yes.